pom.xml

```diff
@@ -1089,14 +1089,8 @@
             </dependency>
             <dependency>
                 <groupId>org.apache.velocity</groupId>
-                <artifactId>velocity</artifactId>
-                <version>1.7</version>
-                <exclusions>
-                    <exclusion>
-                        <groupId>commons-lang</groupId>
-                        <artifactId>commons-lang</artifactId>
-                    </exclusion>
-                </exclusions>
+                <artifactId>velocity-engine-core</artifactId>
+                <version>2.3</version>
```

Comment:
Can we move this version to properties, so that we would be able to set correct version both in root `pom.xml` and in `janusgraph-dist` without setting `2.3` in both files?

Reply:
We Could also just move into dist. As it just a test dependency.